float64 t_TS_ctrl
float64 t_QS_ctrl
float64 t_TS_intpl
float64 t_QS_intpl
string unit
